x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/logical/MvExpand.java

```diff
@@ -27,21 +27,24 @@ public class MvExpand extends UnaryPlan {
 
     private final NamedExpression target;
     private final Attribute expanded;
+    private final Integer limit;
 
     private List<Attribute> output;
 
-    public MvExpand(Source source, LogicalPlan child, NamedExpression target, Attribute expanded) {
+    public MvExpand(Source source, LogicalPlan child, NamedExpression target, Attribute expanded, Integer limit) {
```

Comment:
Since the limit is "artificial" and not serialized, consider keeping the old constructor which delegates to the extended one and just passes limit as null.
Makes the change less disruptive especially in most cases looks like null is passed anyway.